notebooks/features/isolation_forest/IsolationForest - Multivariate Anomaly Detection.ipynb

```diff
@@ -576,7 +629,10 @@
      "nuid": "ff8345cf-fdbc-482d-bf92-5d560724d245",
      "showTitle": false,
      "title": ""
-    }
+    },
+    "tags": [
+     "hide-synapse-internal"
```

Comment:
this notebook is using this alot, is there any way to reduce the usage of these here?

Reply:
Part of this notebook cannot run successfully on Fabric. Will remove the exclusion after I address that issue.